tests/optimizers/gulp/test_uff_assign_ff.py

```diff
@@ -262,19 +203,6 @@ def __init__(self, molecule, atom_types, has_metal, metal_ff):
             has_metal=False,
             metal_ff=None,
         ),
-        # CaseData(
-        #     molecule=stk.BuildingBlock('C1=C(C=C(C=C1C=O)C=O)C=O'),
-        #     atom_types={
-        #         0: 'C_R', 1: 'C_R', 2: 'C_R',
-        #         3: 'C_R', 4: 'C_R', 5: 'C_R',
```

Comment:
Please reinstate these in some way. These are tests that I know will fail based on a known issue.